proctor-common/src/test/java/com/indeed/proctor/common/TestTestRangeSelector.java

```diff
@@ -94,6 +97,17 @@ public void testFindMatchingRule_testDependency() {
                 .isEqualTo(-1);
     }
 
+    @Test
+    public void testFindMatchingRule_Error() {
```

Comment:
Not sure it is worth the effort of mocking and capturing log output for this fix. This is the unit test I used to confirm the allocation rule is now in the error message in console output.